gui-libs/libdecor/libdecor-0.2.2.ebuild

```diff
@@ -0,0 +1,52 @@
+# Copyright 2023-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit meson
+
+DESCRIPTION="A client-side decorations library for Wayland clients"
+HOMEPAGE="https://gitlab.freedesktop.org/libdecor/libdecor"
+if [[ ${PV} == "9999" ]]; then
+	EGIT_REPO_URI="https://gitlab.freedesktop.org/libdecor/libdecor.git"
+	inherit git-r3
+else
+	SRC_URI="https://gitlab.freedesktop.org/libdecor/libdecor/-/archive/${PV}/${P}.tar.bz2"
+	KEYWORDS="~amd64"
+fi
+LICENSE="MIT"
+SLOT="0"
+IUSE="+dbus +gtk examples"
+
+RDEPEND="
+	>=dev-libs/wayland-1.18
+	dbus? ( sys-apps/dbus )
+"
+
+DEPEND="
+	${RDEPEND}
+	>=dev-libs/wayland-protocols-1.15
+	x11-libs/pango
+	x11-libs/cairo
```

Comment:
Did you move pango and cairo? These are definitely both `DEPEND` and `RDEPEND`.

```
$ readelf -d /var/tmp/portage/gui-libs/libdecor-0.2.2/image/usr/lib64/libdecor/plugins-1/libdecor-cairo.so

Dynamic section at offset 0xc260 contains 32 entries:
  Tag        Type                         Name/Value
 0x0000000000000001 (NEEDED)             Shared library: [libdecor-0.so.0]
 0x0000000000000001 (NEEDED)             Shared library: [libwayland-client.so.0]
 0x0000000000000001 (NEEDED)             Shared library: [libpangocairo-1.0.so.0]
 0x0000000000000001 (NEEDED)             Shared library: [libpango-1.0.so.0]
 0x0000000000000001 (NEEDED)             Shared library: [libgobject-2.0.so.0]
 0x0000000000000001 (NEEDED)             Shared library: [libcairo.so.2]
```

Reply:
Thanks for confirming that I wasn't sure that why I put only it `DEPEND`